test/metabase/test/data/generic_sql.clj

```diff
@@ -300,27 +352,40 @@
       (when (seq statement)
         (default-execute-sql! driver context dbdef (s/replace statement #"⅋" ";"))))))
 
-(defn- create-db! [driver {:keys [table-definitions], :as dbdef}]
-  ;; Exec SQL for creating the DB
-  (execute-sql! driver :server dbdef (str (drop-db-if-exists-sql driver dbdef) ";\n"
-                                          (create-db-sql driver dbdef)))
-  ;; Build combined statement for creating tables + FKs
-  (let [statements (atom [])]
-    ;; Add the SQL for creating each Table
+(defn- create-db!
+  ([driver database-definition]
+    (create-db! driver database-definition false))
+  ([driver {:keys [table-definitions], :as dbdef} skip-drop-db]
```

Comment:
normally in Clojure a boolean parameter to a function gets a `?` at the end to mark it as such. `skip-drop-db?` would make more sense here